hamilton/example_slurm_scripts/batchrunner_pipeline_all_classes.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --ntasks 1
+#SBATCH --cpus-per-task 24
+#SBATCH --time=24:0:0
+#SBATCH -J run_mesa_with_batchrunner
+#SBATCH -o run_mesa_with_batchrunner.out
+#SBATCH -e run_mesa_with_batchrunner.err
+# Run on test queue for distributed work
+#SBATCH -p par7.q
+
+module purge
+module load miniconda2/4.1.11
+module load r/4.0.3
+
+source activate classroom_abm
+
+pushd /ddn/home/$USER/classroom-abm/multilevel_analysis
+
+export LD_LIBRARY_PATH=/ddn/home/$USER/usr/lib64
```

Comment:
Can you add a comment here that some people may have this in the /ddn/data directory as that is what the documentation says?

Reply:
I'll change it to match what the docs say - hadn't realised I didn't have it in the same place!